geo/parse.go

```diff
@@ -24,15 +24,24 @@ import (
 // interpret it is as geo point.  Supported formats:
 // Container:
 // slice length 2 (GeoJSON)
-//  first element lon, second element lat
+//
```

Comment:
Why the new empty line here, think you should undo this file's change and simply remove that white space before the text in the comment on line 27, 29, 31, 33 before running `go fmt ./...`